src/istio/control/http/client_context.h

```diff
@@ -55,6 +55,10 @@ class ClientContext : public ClientContextBase {
   // Get the service config cache size
   int service_config_cache_size() const { return service_config_cache_size_; }
 
+  const bool IgnoreForwardedAttributes() const {
```

Comment:
`const bool` is redundant since bool is always const. just use `bool`.

Reply:
style: use snake case to make it consistent with the accessor above.

Reply:
TBH. you don't even need this function since config is accessible via `config()`.